notebook 7.5.0b0

bollwyvl opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

bollwyvl committed:
notebook 7.5.0b0

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.27.05.44.00

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.5

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18098357769](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18098357769). Examine the logs at this URL for more detail.</sub>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18098367869](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18098367869). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
linting

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: failed


Thus the PR was not passing and not merged.

bollwyvl committed:
fix upstream channels

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.27.05.44.00

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.5
